dags/main_summary.py

```diff
@@ -105,15 +105,24 @@
                       dag=dag)
 
 t10 = EMRSparkOperator(task_id="experiments_aggregates",
-                      job_name="Experiments Aggregates View",
-                      execution_timeout=timedelta(hours=10),
-                      instance_count=10,
-                      owner="ssuh@mozilla.com",
-                      email=["telemetry-alerts@mozilla.com", "frank@mozilla.com", "ssuh@mozilla.com"],
-                      env={"date": "{{ ds_nodash }}", "bucket": "{{ task.__class__.private_output_bucket }}"},
-                      uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/experiment_aggregates_view.sh",
-                      dag=dag)
-
+                       job_name="Experiments Aggregates View",
+                       execution_timeout=timedelta(hours=10),
+                       instance_count=10,
+                       owner="ssuh@mozilla.com",
+                       email=["telemetry-alerts@mozilla.com", "frank@mozilla.com", "ssuh@mozilla.com"],
+                       env={"date": "{{ ds_nodash }}", "bucket": "{{ task.__class__.private_output_bucket }}"},
+                       uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/experiment_aggregates_view.sh",
+                       dag=dag)
+
+t11 = EMRSparkOperator(task_id="experiments_aggregates_import",
+                       job_name="Experiments Aggregates Import",
+                       execution_timeout=timedelta(hours=2),
+                       instance_count=1,
+                       owner="chudson@mozilla.com",
+                       email=["telemetry-alerts@mozilla.com", "chudson@mozilla.com"],
+                       env={"date": "{{ ds_nodash }}"},
```

Comment:
We should input the bucket here as well. If we change the output bucket, the `experiments_aggregates` task will write to a new location, which this job should read from.